docs/markdown/Pkgconfig-module.md

```diff
@@ -90,6 +91,9 @@ application. That will cause pkg-config to prefer those
 builddir. This is an experimental feature provided on a best-effort
 basis, it might not work in all use-cases.
 
+*Since 1.9.0* you can specify a license identifier. To use the current project
+licence, simply use `license: meson.project_license()` as argument to `generate()`.
```

Comment:
Should this actually happen by default?

Reply:
I did like that because of the original request:

> It's tempting to have it default to the value provided in the license argument to the project function, but would need to have a way to override, in cases where a project has multiple licenses, but specific libraries are only covered by a subset of those licenses, and perhaps should not automatically add it if a human hasn't verified the values, instead requiring them to add an explicit license: meson.project_license() argument to indicate that they believe that's the correct value.

I do not have strong opinion about this.